tensorflow/core/kernels/mkl_conv_grad_input_ops.cc

```diff
@@ -106,60 +111,79 @@ class MklConvBwdInputPrimitive : public MklPrimitive {
     context_.diff_dst_mem->set_data_handle(
         static_cast<T*>(const_cast<T*>(diff_dst_data)));
 
+#ifdef ENABLE_MKLDNN_V1
+    DCHECK_EQ(context_.bwd_input_primitives.size(),
+              context_.bwd_input_primitives_args.size());
+    for (size_t i = 0; i < context_.bwd_input_primitives.size(); ++i) {
+      context_.bwd_input_primitives.at(i).execute(
+          *context_.bwd_input_stream, context_.bwd_input_primitives_args.at(i));
+    }
+#else
     context_.bwd_input_stream->submit(context_.bwd_input_primitives);
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
Same here. If this gets refactored into a method, please call that method here.

Reply:
In the other PR #36496, the comment says, refactoring can be done later since a bigger redesign is needed. I will also add a TODO here for refactoring when the design will be done.

Reply:
#36496 has been merged. Would you mind making the change in this PR?